config/default/manager_auth_proxy_patch.yaml

```diff
@@ -24,3 +24,4 @@ spec:
         args:
         - "--metrics-addr=127.0.0.1:8080"
         - "--enable-leader-election"
+        - "--contour-image=docker.io/projectcontour/contour:v1.15.1"
```

Comment:
I noticed that there is a v1.16.0 is that not applicable ?

Reply:
I haven't tested it with that particular version of contour. Once the CI is ready it should be easy to test the newer versions. Let's merge this.

Reply:
/lgtm